gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/rollup/RollUpTest.java

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.operators.rollup;
+
+import org.gradoop.flink.model.GradoopFlinkTestBase;
+import org.gradoop.flink.model.api.epgm.GraphCollection;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.model.impl.operators.combination.ReduceCombination;
+import org.gradoop.flink.model.impl.operators.grouping.Grouping;
+import org.gradoop.flink.model.impl.operators.grouping.functions.aggregation.CountAggregator;
+import org.gradoop.flink.model.impl.operators.grouping.functions.aggregation.PropertyValueAggregator;
+import org.gradoop.flink.util.FlinkAsciiGraphLoader;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * A test for {@link RollUp}, calling the operator and checking if the result is
+ * correct.
+ */
+public class RollUpTest extends GradoopFlinkTestBase {
+
+  /**
+   * Executes a rollUp on vertices using a single grouping key and checks if the result
+   * is correct.
+   *
+   * @throws Exception If the execution fails.
+   */
+  @Test
+  public void testVertexRollUpWithSingleGroupingKey() throws Exception {
+    FlinkAsciiGraphLoader loader = getSocialNetworkLoader();
+
+    LogicalGraph input = loader.getGraphCollectionByVariables("g0", "g1", "g2")
+      .reduce(new ReduceCombination());
+
+    //expected
+    loader.initDatabaseFromString("g0:g0 {vertexRollUpGroupingKeys:\"age\"}[" +
+      "(v__0 {count:1L,age:20})" +
```

Comment:
`v0` is also a readable variable. Use find and replace to remove all double underscores if you want. Its only a matter of taste. Your decision.